src/arche/arche.py

```diff
@@ -106,15 +106,15 @@ def schema(self, schema_source):
     def get_items(
         source: Union[str, pd.DataFrame, RawItems],
         count: Optional[int],
-        start: Union[str, int],
+        start: Optional[str],
         filters: Optional[api.Filters],
     ) -> Items:
         if isinstance(source, pd.DataFrame):
             return Items.from_df(source)
         elif isinstance(source, Iterable) and not isinstance(source, str):
-            return Items.from_array(source)
+            return Items.from_array(cast(RawItems, source))
```

Comment:
So without cast it errs because the method also accepts dataframe?
RawItems is `Iterable[Dict[str, Any]]` and here the check is for `Iterable`, isn't it enough?
What if we change the check above?